### PR TITLE
Add posibility to set title

### PR DESCRIPTION
**Pull request by grigorii-horos**
Another attempt to add a title to the box.

Continues from #42

### CODE REVIEW

**Comment by horosgrisa** [comment]
image

File: index.d.ts [hunk 1]
```diff
@@ -73,6 +104,13 @@ declare namespace boxen {
 		*/
 		readonly dimBorder?: boolean;
 
+		/**
+		Reduce opacity of the box title.
+
+		@default false
+		*/
+		readonly dimTitle?: boolean;
```
Comment (sindresorhus) [inline index.d.ts]: I don't really see the point of dimming the title.

File: index.d.ts [hunk 2]
```diff
@@ -34,6 +34,12 @@ declare namespace boxen {
 	}
 
 	interface Options {
+		/**
+		Title of the box.
+		*/
+		readonly title?: string;
+
+
```
Comment (sindresorhus) [inline index.d.ts]: Remove one extra empty line here.

**Comment by sindresorhus** [comment]
You also need to update the readme and add a test.

**Comment by sindresorhus** [comment]
@horosgrisa Bump :)

**Comment by sindresorhus** [comment]
Closing for lack of response.

**Pull request closed without merging**